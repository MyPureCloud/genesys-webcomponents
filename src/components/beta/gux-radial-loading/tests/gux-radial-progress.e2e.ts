import { newE2EPage } from '@stencil/core/testing';

describe('gux-radial-loading', () => {
  [
    '<gux-radial-loading-beta></gux-radial-loading-beta>',
    '<gux-radial-loading-beta context="modal"></gux-radial-loading-beta>',
    '<gux-radial-loading-beta context="input"></gux-radial-loading-beta>',
    '<gux-radial-loading-beta context="full-page"></gux-radial-loading-beta>'
  ].forEach((content, index) => {
    it(`should display component as expected (${index + 1})`, async () => {
      const page = await newE2EPage();

      await page.setContent(content);

      const element = await page.find('gux-radial-loading-beta');

      expect(element.innerHTML).toMatchSnapshot();
    });
  });
});
