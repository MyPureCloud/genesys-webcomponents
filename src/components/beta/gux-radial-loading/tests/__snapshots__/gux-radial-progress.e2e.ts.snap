// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gux-radial-loading should display component as expected (1) 1`] = `"<div class=\\"spinner-container modal\\" aria-busy=\\"true\\"><div class=\\"spin-circle\\"></div></div>"`;

exports[`gux-radial-loading should display component as expected (2) 1`] = `"<div class=\\"spinner-container modal\\" aria-busy=\\"true\\"><div class=\\"spin-circle\\"></div></div>"`;

exports[`gux-radial-loading should display component as expected (3) 1`] = `"<div class=\\"spinner-container input\\" aria-busy=\\"true\\"><div class=\\"spin-circle\\"></div></div>"`;

exports[`gux-radial-loading should display component as expected (4) 1`] = `"<div class=\\"spinner-container full-page\\" aria-busy=\\"true\\"><div class=\\"spin-circle\\"></div></div>"`;
